test/unit_test/options_boost_po_test.cc

```diff
@@ -238,20 +237,6 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(incorrect_option_type_str_for_int, T, option_types
   BOOST_CHECK_THROW(options->add_and_parse(arg_group), VW::vw_argument_invalid_value_exception);
 }
 
-BOOST_AUTO_TEST_CASE(multiple_locations_one_option)
```

Comment:
why remove this test?

Reply:
The new implementations doesn't really care about this situation, whereas boost in the past threw in this case im pretty sure. It's purely a developer side thing

Reply:
is there no value in still keeping the test and not checking for throw, but just checking that it works fine?